components/model.py

```diff
@@ -4,9 +4,10 @@
 '''
 
 import components.database_adapter # database_adaptor.py handles the connection to database
+import uuid
+import hashlib
```

Comment:
I think uuid is an unused import here.
Line 8: standard import "import hashlib" comes before "import components.database_adapter" (wrong-import-order)